web/src/pages/apd/activities/ffp/FedStateSelector.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+import { useFormContext, Controller } from 'react-hook-form';
+import { Dropdown } from '@cmsgov/design-system';
+import PropTypes from 'prop-types';
+import Instruction from '../../../../components/Instruction';
+
+const FedStateSelector = ({ ffp, ffy, setFederalStateSplit }) => {
+  const {
+    control,
+    formState: { errors },
+    setValue
+  } = useFormContext();
+  const [fedStateSplit, setFedStateSplit] = useState(
+    `${ffp.federal}-${ffp.state}`
+  );
+
+  useEffect(() => {
+    setFedStateSplit(`${ffp.federal}-${ffp.state}`);
+  }, [ffp.federal, ffp.state]);
+
+  return (
+    <div className="data-entry-box ds-u-margin-bottom--5">
+      <Instruction
+        source="activities.costAllocate.ffp.federalStateSplitInstruction"
+        headingDisplay={{
+          level: 'p',
+          className: 'ds-h4'
+        }}
+      />
+      <Controller
+        name={`${ffy}.ffp`}
+        control={control}
+        render={({ field: { ...props } }) => (
+          <Dropdown
+            {...props}
+            label="federal-state split"
+            labelClassName="sr-only"
+            options={[
+              { label: 'Select an option', value: '0-100' },
+              { label: '90-10', value: '90-10' },
+              { label: '75-25', value: '75-25' },
+              { label: '50-50', value: '50-50' }
+            ]}
+            value={fedStateSplit}
+            onChange={e => {
```

Comment:
might be cleaner to have all of these function calls wrapped in a handler so it would be something like:
`onChange={handleFedStateSplit(e)}`

Reply:
I'll switch that up